src/Redux/Books/Books.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_BOOK = 'bookstore-cms/books/ADD_BOOK';
+export const REMOVE_BOOK = 'bookstore-cms/books/REMOVE_BOOK';
+export const initialState = [];
+
+/* eslint-disable default-param-last */
```

Comment:
- [ ] Kindly remove this line  & solve the linter manually. As this is going to be mandatory in the next milestone.